app/Support/Shortcode/Shortcode.php

```diff
@@ -232,7 +232,7 @@ private function embedAchievement(int $id): string
 
     private function embedGame(int $id): string
     {
-        $data = Cache::store('array')->rememberForever('game:' . $id . ':card-data', fn () => getGameData($id));
+        $data = Cache::store('array')->rememberForever('game:' . $id . ':game-data', fn () => getGameData($id));
```

Comment:
This makes the key slightly redundant - what is the reasoning for this change?
`game:1:game-data` vs `game:1:card-data` - would it make sense to specify its context here like `game:1:avatar-data`?

Reply:
Without this change, there is a key collision (visible most especially on the cards demo page, which crashes altogether).

The two different keys are fetching different data, so this change seemed reasonable.